packages/core/src/tracing/span.ts

```diff
@@ -626,6 +635,8 @@ export class Span implements SpanInterface {
       trace_id: this._traceId,
       origin: this._attributes[SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN] as SpanOrigin | undefined,
       _metrics_summary: getMetricSummaryJsonForSpan(this),
+      exclusive_time: this._exclusiveTime,
```

Comment:
Can this be sent as an attribute/part of span data instead? We're breaking OTEL compat with this field being top level (ignore metrics summary, we fix it in another way).

Reply:
Removed `exclusive_time` as a field from spans. If we're passing it in `data`, then no changes are needed to support that. This pr is just for `measurements` now. Let me know if this sounds good!